### PR TITLE
Modify user guide for add command

### PR DESCRIPTION
**Pull request by kailashgautham**

### CODE REVIEW

**Review by teojunda — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -75,20 +75,15 @@ Shows a message explaning how to access the help page.
...
-
-**Tip:** A person can have any number of tags (including 0)
-</box>
+Format: `ac n/NAME c/CLASS_GROUP e/EMAIL p/PHONE_NUMBER [t/TELEGRAM] [g/GITHUB]`
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `ac n/John Doe c/F14-3 p/98765432 e/johnd@example.com t/@johndoe g/johnDoeGithub`
+* `add n/Betsy Crowe c/T10-2 e/betsycrowe@example.com p/1234567 g/betsycodes`
```
Comment (teojunda) [inline docs/UserGuide.md:86]: I think you missed the add in this line.

Comment (kailashgautham) [reply docs/UserGuide.md]: Oop good catch! Missed it in a couple other places as well, I've fixed it now.

**Review by hiivan — APPROVED** [review]
LGTM Kailash!

**Pull request merged**